include/fmt/std.h

```diff
@@ -435,6 +437,28 @@ struct formatter<BitRef, Char,
   }
 };
 
-FMT_END_NAMESPACE
+FMT_EXPORT
+template <typename T, typename Char>
+struct formatter<std::atomic<T>, Char> : formatter<T, Char> {
```

Comment:
This formatter should only be enabled if T is formattable.

Reply:
Fixed